packages/icons/icons.json

```
@@ -373,91 +373,91 @@
     {
         "displayName": "Import",
         "iconName": "import",
-        "tags": "arrow, down, importing,",
+        "tags": "arrow, down, importing, , download",
         "group": "action",
         "codepoint": 61946
     },
     {
         "displayName": "Export",
         "iconName": "export",
-        "tags": "arrow, up, exporting",
+        "tags": "arrow, up, exporting, output, move out, upload, send",
         "group": "action",
         "codepoint": 61856
     },
     {
         "displayName": "Minimize",
         "iconName": "minimize",
-        "tags": "arrows, decrease, smaller",
+        "tags": "arrows, decrease, smaller, combine, shrink, compress, together, crash, resize, collision, close, connect, merge, conflict",
         "group": "action",
         "codepoint": 62019
     },
     {
         "displayName": "Maximize",
         "iconName": "maximize",
-        "tags": "arrows, increase, bigger",
+        "tags": "arrows, increase, bigger, expand, opposite diagonal arrows, diagonal bidirectional arrows, double diagonal arrows",
         "group": "action",
         "codepoint": 62012
     },
     {
         "displayName": "Tick",
         "iconName": "tick",
-        "tags": "mark, sign, ok, approved, success",
+        "tags": "mark, sign, ok, approved, success, check, verizon, done, complete, confirm",
         "group": "action",
         "codepoint": 62191
     },
     {
         "displayName": "Cross",
         "iconName": "cross",
-        "tags": "cross mark, fail, delete, no, close, remove",
+        "tags": "cross mark, fail, delete, no, close, remove, not available, wrong/incorrect, cancel, x",
         "group": "action",
         "codepoint": 61801
     },
     {
         "displayName": "Plus",
         "iconName": "plus",
-        "tags": "sign, add, maximize, zoom in",
+        "tags": "sign, add, maximize, zoom in, add or append, enlarge or zoom-in, increase",
         "group": "action",
         "codepoint": 62073
     },
     {
         "displayName": "Minus",
         "iconName": "minus",
-        "tags": "sign, remove, minimize, zoom out",
+        "tags": "sign, remove, minimize, zoom out, line, dash, space, hyphen, emdash, horizonal thin line",
         "group": "action",
         "codepoint": 62020
     },
     {
         "displayName": "Arrow: left",
         "iconName": "arrow-left",
-        "tags": "arrow, direction, left",
+        "tags": "arrow, direction, left, left arrow, backspace, undo, back",
```

```suggestion
        "tags": "arrow, direction, left, backspace, back",
```

removed a redundant tag. also we already have an "undo" icon, I don't want to conflate them